### ISSUE TITLE
Plotly vs Mayavi

### ISSUE THREAD

**Issue by jswoboda** [jswoboda/GeoDataPython#7]
I've noticed that Mayavi has a rather slow development cycle. In order to use it I have to depreciate my matplotlib version to 1.5. Plotly seem a little bit better supported and can do most of the [3-D plotting](https://nbviewer.jupyter.org/github/empet/Plotly-plots/blob/master/Plotly-Slice-in-volumetric-data.ipynb) that's needed. I may branch off some experimental code to compare. As seen from the link it seems to work well with the ipython notebooks.

**Comment by scivision** [comment]
Yes and you can use Plotly totally offline, and Plotly is much easier to install, and can be used from plain Python or Ipython
https://github.com/scivision/plotly3d-examples-python

**Comment by jswoboda** [comment]
I've made a slice command that uses matplotlib. Its not great but it does work and I'll be pushing it up. I'm also planning on moving the mayvi plotting to a different file, plotingmayvi.py. I also want to explore using plotly as well but this will also be a separate file to keep it organized. I will update the readme accordingly to point out these changes.

**Comment by scivision** [comment]
I think that would be good, to have separate files for Plotly and Mayavi, as they require unique imports and are optional. Mayavi is integrating updates to bring QT5, but unknown how long this will take.
